README.md

```diff
@@ -8,21 +8,214 @@
 
 ## Overview
```

Comment:
Can you restructure the `Overview` section by using following as a reference. 

https://github.com/ballerina-platform/module-ballerinax-hubspot.crm.object.deals/blob/d9c9c839279da32f429477537ccd250b399f02af/README.md#overview